dashboard-for-desktop/articles/dashboard-designer/designing-dashboard-items/cards/conditional-formatting.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Conditional Formatting
+author: Margarita Zakhodyaeva
+---
+
+# Conditional Formatting
+
+For the Card dashboard item, you can apply conditional formatting to the card's visual elements (like Title, Subtitle, different values) and change the card's background.
```

Comment:
What do you mean by "different values"?

Reply:
actual, target, different types of delta values (like percent variation).

Reply:
```suggestion
For a Card dashboard item, you can apply conditional formatting to the card's visual elements (e.g., Title, Subtitle, various values) and change the card's background.
```